winlogbeat/eventlog/wineventlog_test.go

```diff
@@ -167,6 +166,20 @@ func createLog(t testing.TB, messageFiles ...string) (log *eventlog.Log, tearDow
 	return log, tearDown
 }
 
+func safeWriteEvent(t testing.TB, log *eventlog.Log, etype uint16, eid uint32, msgs []string) {
+	deadline := time.Now().Add(time.Second * 10)
+	for {
+		err := log.Report(etype, eid, msgs)
```

Comment:
Curious what the story is behind this. What kind of temporary errors were you seeing?

Reply:
You get an `access denied` error if the events are published too shortly after creating the event log